.env

```diff
@@ -5,25 +5,25 @@ REACT_APP_TRAININGS_URL=
 REACT_APP_DNS=
 REACT_APP_GIT_PATH=
 REACT_APP_PASSWORD=
-REACT_APP_GRAFANA_BASE_URI=
+REACT_APP_MONITORING_BASE_URI=
 # MINIO
 REACT_APP_MINIO_BASE_URI=
 REACT_APP_MINIO_END_POINT=
 REACT_APP_MINIO_PORT=
 REACT_APP_MINIO_END_MINIMUM_DURATION_MS=36000000
 # END POINTS
-REACT_APP_SALON_ROCKETCHAT=
+REACT_APP_SALON_CHAT=
```

Comment:
Salon seems to be a french only word. Moving from rocketchat to chat is good but we could take this opportunity to rename it further